assets/javascripts/running.js

```diff
@@ -321,18 +321,29 @@ var last_event;
 function loadCanvas(canvas, dataURL) {
   var context = canvas.getContext('2d');
 
-  // load image from data url
+  // Load image from data URL
   var scrn = new Image();
   scrn.onload = function () {
     canvas.width = this.width;
     canvas.height = this.height;
     context.clearRect(0, 0, this.width, this.width);
     context.drawImage(this, 0, 0);
+
+    // Hide loading animation after the first image is loaded
+    var loadingElement = document.getElementById('liveview-loading');
+    if (loadingElement) {
+      loadingElement.style.display = 'none';
+    }
   };
   scrn.src = dataURL;
 }
 
 function initLivestream() {
+  // Show the loading animation when initializing the live stream
+  var loadingElement = document.getElementById('liveview-loading');
+  if (loadingElement) {
+    loadingElement.style.display = 'block';
+  }
```

Comment:
Unless you show me in a test I don't believe this should be necessary at all. Proof please :)